app/api/grape_api/user_api.rb

```diff
@@ -0,0 +1,191 @@
+module GrapeAPI
```

Comment:
Two things:

- There's functionality already existing in `app/controllers/application_tokens_controller.rb` for this. Could we merge the code somehow ? (either changes the frontend to use this API instead of the controller, thus removing the controller, or creating methods that both the controller and the API use)
- Could you add more documentation ?